src/blocks/extensions/advanced-settings/index.js

```diff
@@ -137,7 +138,7 @@ const UserConditionOptions = ( props ) => {
 									value={ o.value }
 									sunday
 									onChange={ handleChange }
-									checked={ UAGDay.includes( o.value ) ? true : false }
+									checked={ UAGDay?.includes( o.value ) ? true : false }
```

Comment:
Wouldn't this just be if `UAGDay` has an includes method, and if it does include o.value, then check else not?

I feel we can simplify this with the Boolean function.
Try it out and ensure that it works as needed.

```suggestion
									checked={ Boolean( UAGDay?.includes( o.value ) ) }
```

Reply:
Its not working as expected keeping it same